app/models/entity.rb

```diff
@@ -0,0 +1,4 @@
+class Entity < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :category, class_name: 'Group', foreign_key: :category_id
```

Comment:
- You did well so far, but please note that you have not properly set up the relationship between your models in the database according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/raw/main/capstone/images/erd_diagram.png). The relationship between the `Entity` and `Group` model should be `many-many`. Please fix this 🙏